firmware/src/rdm.h

```diff
@@ -265,6 +265,14 @@ typedef enum {
 
 
 /**
+ * @brief Open Lighting Project manufacturer PIDs.
+ */
+typedef enum {
+  PID_DEVICE_MODEL = 0x8002,
+  PID_DEVICE_MODEL_LIST = 0x8003
+} OpenLightingManufacturerPID;
```

Comment:
Add an alternative DEVICE_MODEL_DESCRIPTION/LABEL to get descriptions without having to switch into the mode.